docs/docs/api/endpoints/repositories.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Repositories
+sidebar_label: Repositories
+---
+
+The `repositories.*` API is about controlling the repositories from where the individual components are installed.
```

Comment:
Since we agreed to not do this in the scope of this milestone I'd suggest we hide this from the sidebar or mark it as a draft

Reply:
But if the open source repository is hard-coded as a default we don't need that two step approach, right?